pkgs/development/libraries/newt/default.nix

```diff
@@ -1,4 +1,6 @@
-{ fetchurl, stdenv, slang, popt }:
+{ lib, fetchurl, stdenv, slang, popt
+, python ? null, pythonSupport ? false
```

Comment:
defaulting `python` to null seems dangerous to me, as if I override to set `pythonSupport` to true I'll get confusing error messages without passing it. Having `python` in here but only conditionally used seems like it'd be fine to me.